proto/frequenz/api/electricity_trading/v1/electricity_trading.proto

```diff
@@ -421,46 +422,106 @@ message PublicTrade {
   uint64 id = 1;
 
   // Delivery area code of the buy side.
-  frequenz.api.common.v1.grid.DeliveryArea buy_delivery_area = 2;
+  frequenz.api.common.v1alpha7.grid.DeliveryArea buy_delivery_area = 2;
 
   // Delivery area code of the sell side.
-  frequenz.api.common.v1.grid.DeliveryArea sell_delivery_area = 3;
+  frequenz.api.common.v1alpha7.grid.DeliveryArea sell_delivery_area = 3;
 
   // The delivery period for the contract.
-  frequenz.api.common.v1.grid.DeliveryPeriod delivery_period = 4;
+  frequenz.api.common.v1alpha7.grid.DeliveryPeriod delivery_period = 4;
 
   // UTC Timestamp of the trades execution time.
   google.protobuf.Timestamp execution_time = 5;
 
   // The price at which the trade was executed.
-  frequenz.api.common.v1.market.Price price = 6;
+  frequenz.api.common.v1alpha7.market.Price price = 6;
 
   // The executed quantity of the contract traded.
-  frequenz.api.common.v1.market.Power quantity = 7;
+  frequenz.api.common.v1alpha7.market.Power quantity = 7;
 
   // Final state of the trade.
   TradeState state = 8;
 }
 
-// Parameters for filtering Gridpool orders.
+// Filters delivery periods by start time and duration.
+//
+// A delivery period is defined by a start timestamp and a fixed delivery
+// duration (e.g., 15, 30, or 60 minutes). This filter lets you query for
+// delivery periods that start within an optional time range and optionally
+// match one or more specific durations.
+//
+// The time range is inclusive at the start and exclusive at the end, following
+// the ISO 8601 interval format and conventions in tools like Pandas and Python.
+//
+// !!! tip "Terminology"
+//     - **DeliveryDuration**: A fixed interval (e.g., 15, 30, or 60 minutes).
+//     - **DeliveryPeriod**: A combination of a start time and a
+//                           DeliveryDuration.
+//
+// !!! note "Filter Logic"
+//     - `time_interval` is optional; if unset, all time ranges are matched.
+//     - `duration_filters` is optional. If unset or empty, all durations will
```

Comment:
Maybe `omitted` instead of unset to be consistent with the following notes/examples?
```suggestion
//     - `time_interval` is optional; if omitted, all time ranges are matched.
//     - `duration_filters` is optional. If omitted or empty, all durations will
```